src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: $day-font;
+  font-size: $day-text-size;
```

Comment:
The variable `$day-text-size` is used here, but it is not defined in this file. Ensure it is defined in a file that is imported before this one, or define it here.